client/stylesheets/_responsive.scss

```diff
@@ -344,6 +424,14 @@
         margin-top: 20px !important;
     }
 
+    .homeCardLi {
+        width: 130px;
+        margin: 6px 4px !important;
+    }
+
+    .homeCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `homeCardImg` should be written in lowercase with hyphens